tests/EventLoopSpec.hs

```diff
@@ -62,7 +62,7 @@ masterBranch = BaseBranch "master"
 -- Invokes Git with the given arguments, returns its stdout. Crashes if invoking
 -- Git failed. Discards all logging.
```

Comment:
The comment on top of this function is now incorrect. We do not discard all logging we explicitly raise it using `error`.

Reply:
We do discard some logging though (that's `fakeRunLogger`). The `Left` branch here is just exit code & output if git fails.

Reply:
Ah cool missed that part!